ckanext/ontario_theme/fanstatic/ontario_theme.less

```diff
@@ -1,5 +1,25 @@
 @import "smarties.less";
```

Comment:
Looks like the compiled less files into css were missed. Can you add this and resubmit?